[2022-11-21T01:03:50.402+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2022-11-21T01:03:50.419+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2022-11-21T01:03:50.421+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-21T01:03:50.422+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-21T01:03:50.423+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-21T01:03:50.478+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): http_check_copy> on 2022-11-20 00:00:00+00:00
[2022-11-21T01:03:50.489+0000] {standard_task_runner.py:55} INFO - Started process 33719 to run task
[2022-11-21T01:03:50.507+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cp_webcheck_failed_copy', 'http_check_copy', 'scheduled__2022-11-20T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/cp_webcheck_failed copy.py', '--cfg-path', '/tmp/tmp6hhmmavu']
[2022-11-21T01:03:50.510+0000] {standard_task_runner.py:83} INFO - Job 66: Subtask http_check_copy
[2022-11-21T01:03:50.614+0000] {task_command.py:376} INFO - Running <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-20T00:00:00+00:00 [running]> on host 71a129efd3d8
[2022-11-21T01:03:50.726+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cp_webcheck_failed_copy
AIRFLOW_CTX_TASK_ID=http_check_copy
AIRFLOW_CTX_EXECUTION_DATE=2022-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-20T00:00:00+00:00
[2022-11-21T01:03:50.733+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-21T01:03:50.735+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -s https://localhost > /dev/null']
[2022-11-21T01:03:50.776+0000] {subprocess.py:86} INFO - Output:
[2022-11-21T01:03:50.798+0000] {subprocess.py:97} INFO - Command exited with return code 7
[2022-11-21T01:03:50.814+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 7.
[2022-11-21T01:03:50.823+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=cp_webcheck_failed_copy, task_id=http_check_copy, execution_date=20221120T000000, start_date=20221121T010350, end_date=20221121T010350
[2022-11-21T01:03:50.843+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 66 for task http_check_copy (Bash command failed. The command returned a non-zero exit code 7.; 33719)
[2022-11-21T01:03:50.876+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-11-21T01:03:50.896+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
